cmudb/tscout/tscout.py

```diff
@@ -202,11 +204,16 @@ def lost_something(num_lost):
     pass
 
 
-def processor(ou, buffered_strings):
-    setproctitle.setproctitle("{} Processor".format(ou.name()))
+
+def processor(ou, buffered_strings, outdir=None):
```

Comment:
oh, and if you incorporate the below proposal, outdir should be a required argument now